components/GameStatusButton.vue

```diff
@@ -45,13 +100,32 @@ import {
   QueueListIcon,
   TrashIcon,
   WrenchIcon,
+  InformationCircleIcon,
 } from "@heroicons/vue/20/solid";
+import { Dialog, DialogPanel, DialogTitle, TransitionChild, TransitionRoot } from '@headlessui/vue'
 
 import type { Component } from "vue";
 import { GameStatusEnum, type GameStatus } from "~/types.js";
 import { Menu, MenuButton, MenuItem, MenuItems } from '@headlessui/vue'
+import { useRoute } from 'vue-router';
+import { invoke } from "@tauri-apps/api/core";
+
+const props = defineProps<{ 
+  status: GameStatus;
+  gameId?: string;
+}>();
+
+const route = useRoute();
+const displayGameId = computed(() => props.gameId || route.params.id as string);
```

Comment:
Another reason why this shouldn't include this dropdown - GameStatusButton should *just* be aware of the game status, nothing else.

Reply:
Did you commit the change that adds the `:game-id` prop on the pages/library/[id]/index.vue side?